deploy/gluster-block/kubernetes/csi-deployment.yaml

```diff
@@ -0,0 +1,287 @@
+# Copyright 2018 The Gluster CSI Authors.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#    http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+kind: Service
+apiVersion: v1
+metadata:
+  name: csi-attacher-glusterblockplugin
+  labels:
+    app: csi-attacher-glusterblockplugin
+spec:
+  selector:
+    app: csi-attacher-glusterblockplugin
+  ports:
+    - name: dummy
+      port: 12345
+
+---
+kind: StatefulSet
+apiVersion: apps/v1beta1
+metadata:
+  name: csi-attacher-glusterblockplugin
+spec:
+  serviceName: "csi-attacher-glusterblockplugin"
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: csi-attacher-glusterblockplugin
+    spec:
+      serviceAccount: csi-attacher
+      containers:
+        - name: csi-attacher
+          image: quay.io/k8scsi/csi-attacher:v0.4.0
+          args:
+            - "--v=5"
+            - "--csi-address=$(ADDRESS)"
+          env:
+            - name: ADDRESS
+              value: /var/lib/csi/sockets/pluginproxy/csi.sock
+          imagePullPolicy: "IfNotPresent"
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /var/lib/csi/sockets/pluginproxy/
+
+        - name: glusterblock
+          image: docker.io/poornimag/glusterblock-csi-driver
+          args:
+            - "--nodeid=$(NODE_ID)"
+            - "--endpoint=$(CSI_ENDPOINT)"
+            - "--resturl=$(REST_URL)"
```

Comment:
as of now, we are not communicating with gd2. I think we can remove this argument

Reply:
this is applicable in all places